test/selector.jl

```diff
@@ -38,6 +38,24 @@ a = [1 2  3  4
         @test view(da, Between((11, 20)), Between((2u"s", 3u"s"))) == [6 7]
     end
 
+    @testset "mixed selectors and standard" begin
+        indices = [
+            (Between(9, 31), Near((3:4)u"s")),
+            (Near(22), At([3.0u"s", 4.0u"s"])),
+            (At(20), At((2:3)u"s")),
+            (Near<|13, Near<|[1.3u"s", 3.3u"s"]),
```

Comment:
These all seem to be selectors?

Shouldn't you have `:`, `2`, `1:10` etc in here as well?

Reply:
I convert these to their equivalent indexing statements with `sel2indices` below in lines 51:53.

Reply:
Maybe make it more explicit and just write it out - it doesn't really communicate what is being tested here.

Reply:
Would this be more clear?

```julia
equivalent_cases = product(sel_pair, sel2indices(da, sel_pair))
for (case1, case2) in combinations(equivalent_cases, 2)
    # Tests that the cases are equivalent
end
```

Reply:
These are the same test cases as are used in the `"selectors without dim wrappers"` tests.

My preference is to seperate the test data from the code being tested so it's easy to have new data go through all checks, or replace the cases with a case generator.

Reply:
I don't want to use sel2indices in the tests like that. Its too coupled and not clear to people reading them what you intend to test. Just write them out.